custom_components/automower/const.py

```diff
@@ -14,69 +14,140 @@
 DEFAULT_ICON = "mdi:robot-mower"
 VENDOR = "Husqvarna"
 VACUUM_SCAN_INTERVAL = 60
-VERSION = "0.5.1"
+VERSION = "0.5.2"
 
 # TODO: Add more statuses as we observe them
-STATUS_ERROR = "ERROR"
-STATUS_OK_CHARGING = "OK_CHARGING"
-STATUS_OK_CUTTING = "OK_CUTTING"
-STATUS_OK_CUTTING_MANUAL = "OK_CUTTING_NOT_AUTO"
-STATUS_OK_LEAVING = "OK_LEAVING"
-STATUS_OK_SEARCHING = "OK_SEARCHING"
-STATUS_PARKED_TIMER = "PARKED_TIMER"
-STATUS_PARKED_AUTOTIMER = "PARKED_AUTOTIMER"
-STATUS_PARKED_PARKED_SELECTED = "PARKED_PARKED_SELECTED"
-STATUS_PAUSED = "PAUSED"
-STATUS_EXECUTING_PARK = "EXECUTING_PARK"
-STATUS_EXECUTING_START = "EXECUTING_START"
-STATUS_EXECUTING_STOP = "EXECUTING_STOP"
-STATUS_OFF_HATCH_OPEN = "OFF_HATCH_OPEN"
-STATUS_OFF_HATCH_CLOSED = "OFF_HATCH_CLOSED_DISABLED"
-STATUS_OFF_DISABLED = "OFF_DISABLED"
+STATUS_ERROR =                  'ERROR'
+STATUS_OK_CHARGING =            'OK_CHARGING'
+STATUS_OK_CUTTING =             'OK_CUTTING'
+STATUS_OK_CUTTING_MANUAL =      'OK_CUTTING_NOT_AUTO'
+STATUS_OK_LEAVING =             'OK_LEAVING'
+STATUS_OK_SEARCHING =           'OK_SEARCHING'
+STATUS_PARKED_TIMER =           'PARKED_TIMER'
+STATUS_PARKED_AUTOTIMER =       'PARKED_AUTOTIMER'
+STATUS_PARKED_PARKED_SELECTED = 'PARKED_PARKED_SELECTED'
+STATUS_PAUSED =                 'PAUSED'
+STATUS_EXECUTING_PARK =         'EXECUTING_PARK'
+STATUS_EXECUTING_START =        'EXECUTING_START'
+STATUS_EXECUTING_STOP =         'EXECUTING_STOP'
+STATUS_WAIT_POWER_UP =          'WAIT_POWER_UP'
+STATUS_OFF_HATCH_OPEN =         'OFF_HATCH_OPEN'
+STATUS_OFF_HATCH_CLOSED =       'OFF_HATCH_CLOSED_DISABLED'
+STATUS_OFF_DISABLED =           'OFF_DISABLED'
```

Comment:
The code should be formatted with black, in the same way as the whole HA code-base. I agree that your suggestion is more legible, but black wins.